doc/classes/VisualShaderNodeComment.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="VisualShaderNodeComment" inherits="VisualShaderNodeFrame" deprecated="This class has no function anymore and only exists for compatibility." xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Only exists for compatibility. Behaves exactly like [VisualShaderNodeFrame].
+	</brief_description>
+	<description>
+	</description>
```

Comment:
We should figure out something to write here too just to keep the completion ratio high in https://godotengine.github.io/doc-status/ :)